pandas/tests/indexing/test_loc.py

```diff
@@ -427,6 +427,13 @@ def test_loc_setitem_consistency(self):
         expected = DataFrame({'date': Series(['string'])})
         tm.assert_frame_equal(df, expected)
 
```

Comment:
can you move this to pandas/tests/frame/test_indexing.py

make it a new test. (with _datetime in the name). can you add all of the examples in the original post?